packages/ui/src/lib/index.js

```diff
@@ -13,9 +13,10 @@ export { default as RadioButtonGroup } from './radioButton/RadioButtonGroup.svel
 export { default as RadioButtonSolid } from './radioButtonSolid/RadioButtonSolid.svelte';
 export { default as RadioButtonGroupSolid } from './radioButtonSolid/RadioButtonGroupSolid.svelte';
 export { default as Select } from './select/Select.svelte';
+export { default as Spinner } from './spinners/Spinner.svelte'
 export { default as TabbedSidebar } from './tabbedSidebar/TabbedSidebar.svelte';
 export { default as TabbedSidebarTabLabel } from './tabbedSidebar/TabbedSidebarTabLabel.svelte';
 export { default as TabbedSidebarTabList } from './tabbedSidebar/TabbedSidebarTabList.svelte';
 export { default as TabbedSidebarWrapper } from './tabbedSidebar/TabbedSidebarWrapper.svelte';
 export { default as TabLabel } from './tabs/TabLabel.svelte';
-export { default as TabList } from './tabs/TabList.svelte';
+export { default as TabList } from './tabs/TabList.svelte';
```

Comment:
```suggestion
export { default as TabList } from './tabs/TabList.svelte';

```

Prettier wants a blank line at the end of this file.